lib/SIL/SILFunctionType.cpp

```diff
@@ -1777,7 +1777,7 @@ static bool isImporterGeneratedAccessor(const clang::Decl *clangDecl,
     return false;
 
   // Must be a type member.
-  if (constant.getUncurryLevel() != 1)
+  if (constant.getParameterListCount() != 2)
```

Comment:
Can you add a comment here why you are checking against 2? Couldn't this be a method on SILDeclRef with a nice comment that explains what is happening here?